packages/headless/docs.config.json

```diff
@@ -0,0 +1,119 @@
+{
+  "engine": {
+    "sources": [
+      "src/app/headless-engine"
+    ]
+  },
+  "actions": {
+    "sections": [
+      {
+        "name": "Configuration",
+        "sources": [
+          "src/features/configuration/configuration-actions"
+        ]
+      },
+      {
+        "name": "Did-You-Mean",
+        "sources": [
+          "src/features/did-you-mean/did-you-mean-actions"
+        ]
+      }
+    ]
+  },
+  "controllers": [
+    {
+      "name": "BreadcrumbManager",
+      "sources": ["src/controllers/breadcrumb-manager/"]
+    },
+    {
+      "name": "Context",
+      "sources": ["src/controllers/context/"]
+    },
+    {
+      "name": "DidYouMean",
+      "sources": ["src/controllers/did-you-mean/"]
+    },
+    {
+      "name": "FacetManager",
+      "sources": ["src/controllers/facet-manager/"]
+    },
+    {
+      "name": "CategoryFacet",
+      "sources": ["src/controllers/facets/category-facet/"]
+    },
+    {
+      "name": "Facet",
+      "sources": ["src/controllers/facets/facet/"]
+    },
+    {
+      "name": "FacetSearch",
+      "sources": ["src/controllers/facets/facet-search/specific/"]
+    },
+    {
+      "name": "CategoryFacetSearch",
+      "sources": ["src/controllers/facets/facet-search/category/"]
```

Comment:
The `facetSearch` and `CategoryFacetSearch` controllers are not exported, but are bundled with the `Facet` and `CategoryFacet` controllers. I would place their docs on the same page if possible,